src/compiler/compile/render_ssr/handlers/Element.ts

```diff
@@ -113,7 +113,16 @@ export default function(node: Element, renderer: Renderer, options: RenderOption
 		}
 
 		if (name === 'group') {
-			// TODO server-render group bindings
+			const value_attribute = node.attributes.find(({ name }) => name === 'value');
+			if (value_attribute && value_attribute.chunks.length === 1) {
+				const value = get_attribute_value(value_attribute);
+				const bound = expression.node;
+				renderer.add_expression(x`
+					${value} === ${bound} || Array.isArray(${bound}) && ${bound}.includes(${value})
```

Comment:
Why are you allowing `${value} === ${bound}` instead of just checking whether it's an array that includes this value?

Reply:
It's for the mutually exclusive groups where the bound value is not an array but a string:
```svelte
<script>
	let tortilla = 'Plain';
</script>

<!-- grouped radio inputs are mutually exclusive -->
<input type="radio" bind:group={tortilla} value="Plain">
<input type="radio" bind:group={tortilla} value="Whole wheat">
<input type="radio" bind:group={tortilla} value="Spinach">
```

With the above inputs it would do the following checks:
```js
'Plain' === 'Plain'  // checked
'Whole wheat' === 'Plain'
'Spinach' === 'Plain'
```